rust-bot/src/commands/mod_group.rs

```diff
@@ -19,12 +19,41 @@ use serenity::{
 
 #[group]
 #[checks(Mod)]
-#[commands(create_cohort)]
+#[commands(add_role_to_users, create_cohort)]
 struct Mod;
 
 // Party Corgi - Mod Role Id = 639531892437286959
 const MOD_ROLE_ID: u64 = 639531892437286959;
 
+#[command]
+#[aliases("add_role", "add-role")]
+fn add_role_to_users(ctx: &mut Context, msg: &Message, mut args: Args) -> CommandResult {
+    if (args.len() as u32) < 2 {
+        error!(err = "Didn't pass enough arguments to add_role");
+        msg.reply(&ctx.http, "You don't have enough args!").unwrap();
+    } else {
+        let role_id = args.single::<RoleId>().unwrap();
```

Comment:
should we be using ? instead of these unwraps?

Reply:
Yeah it might work better if we do that instead, good call!

Reply:
For `msg.reply` we could also do `.unwrap_or_else(|err| error!(err = error))` to send the reply error to Honeycomb.